Random_Problems/4Sum.cpp

```diff
@@ -11,14 +25,15 @@ class Solution {
                 int start = j+1,end = n-1;
                 while(start<end){
                     long long crSum = (long long)nums[i]+nums[j]+nums[start]+nums[end];
-                    if(crSum == target){
+                    if(crSum == target)
```

Comment:
you can go with either of this
`if (condition) {
}`
or 
`if(condition)` 
`{`
`  some code`
`}`
but be consistent